mlir/test/Catalyst/BufferizationTest.mlir

```diff
@@ -45,3 +45,17 @@ func.func @dbprint_str() {
 
     return
 }
+
+// -----
+
+func.func @custom_call(%arg0: tensor<3x3xf64>) -> tensor<3x3xf64> {
+    // CHECK: [[memrefArg:%.+]] = bufferization.to_memref %arg0 : memref<3x3xf64>
+    // CHECK: [[alloc:%.+]] = bufferization.alloc_tensor() {bufferization.escape = [false], memory_space = 0 : i64} : tensor<3x3xf64>
```

Comment:
Ideally I would not capture such details, unless we explicitly intend for them to be there.
```suggestion
    // CHECK: [[alloc:%.+]] = bufferization.alloc_tensor() {{.*}}: tensor<3x3xf64>
```